test/pred-sets/ref/ccb_implicit_and_explicit_interactions.inv

```diff
@@ -8,7 +8,7 @@ lda:0
 0 skip:
 options: --cb_adf --cb_explore_adf --cb_sample --cb_type mtr --ccb_explore_adf --csoaa_ldf multiline --csoaa_rank --interactions AAA --interactions ::: --quadratic :: --quadratic AB
 Checksum: 2813477556
-CCB: has_seen_multi_slot_example = 1
+CCB: has_seen_multi_slot_example = true
```

Comment:
can this sort of change bite us back when we read them? this mismatch across versions, i.e. reading and old readeable model with this new impl?

Reply:
Readable models cannot be parsed by VW, the are an output only thing.